src/mscorlib/model.xml

```diff
@@ -7389,7 +7389,7 @@
     </Type>
     <Type Name="System.Threading.ThreadStartException">
       <Member Status="ImplRoot" Name="#ctor(System.Exception)" /><!-- EE -->
-      <Member Name="#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)" />
+      <Member Status="ImplRoot" Name="#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)" />
```

Comment:
> but do we need it in this case?

The member is internal:
https://github.com/dotnet/coreclr/blob/dbe69102224886888874b56f5e2c5b3905bd8fa2/src/mscorlib/src/System/Threading/ThreadStartException.cs#L29